Libraries/src/Amazon.Lambda.Serialization.Json/AwsResolver.cs

```diff
@@ -86,7 +86,7 @@ protected override IList<JsonProperty> CreateProperties(Type type, MemberSeriali
                     }
                 }
             }
-            else if (type.FullName.Equals("Amazon.DynamoDBv2.Model.StreamRecord", StringComparison.Ordinal))

```

Comment:
You will also need to bump the version number for this project in the csproj file.

Reply:
Updated the version in https://github.com/aws/aws-lambda-dotnet/commit/3bd10bc4900e4d21ebba4d47fe9700cd0076a74a#diff-87dd4211c373a801da796dc71249823e727ca7e71d70f0a2f87bac50b1288244R12

I didn't do a major version bump there, please let me know if that was needed.